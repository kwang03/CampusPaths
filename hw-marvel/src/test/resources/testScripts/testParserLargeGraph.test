#Tests the parser on a graph with 5 characters

LoadGraph myGraph largeGraph.csv

ListNodes myGraph

ListChildren myGraph Char1

ListChildren myGraph Char2

ListChildren myGraph Char3

ListChildren myGraph Char4

ListChildren myGraph Char5