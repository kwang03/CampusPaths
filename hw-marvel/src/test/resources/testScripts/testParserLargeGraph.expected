#Tests the parser on a graph with 5 characters

loaded graph myGraph

myGraph contains: Char1 Char2 Char3 Char4 Char5

the children of Char1 in myGraph are: Char2(Book1) Char5(Book1)

the children of Char2 in myGraph are: Char1(Book1) Char4(Book2) Char5(Book1)

the children of Char3 in myGraph are:

the children of Char4 in myGraph are: Char2(Book2)

the children of Char5 in myGraph are: Char1(Book1) Char2(Book1)